### ISSUE TITLE
Documentation for Vite is outdated

### ISSUE THREAD

**Issue by RPDeshaies** [single-spa/single-spa.js.org#460]
**Describe the bug or question**

The documentation on how to integrate Single Spa with ViteJS is outdated. 

https://single-spa.js.org/docs/ecosystem-vite/

The examples laid out on the page areonly compatible with ViteJS v1 and not the most recent v2.3.5.

There's a v2 migration guide available [here](https://vitejs.dev/guide/migration.html#manifest-format-change), and I was able to convert most of the config to v2, though I don't know what to do with the `configureServer` property that doesn't seem to exist in v2.

```tsx
export default defineConfig({
    build: {
        rollupOptions: {
            input: 'src/index.tsx',
            preserveEntrySignatures: 'exports-only',
            output: {
                format: 'systemjs',
            },
        },
    },
});

```

**Comment by internettrans** [comment]
Thanks for the bug report, I've transferred this to single-spa.js.org repo. The vite example repo shows usage with Vite 2, just the documentation needs to match.

**Comment by internettrans** [comment]
Resolved in #461
